rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -60,57 +70,58 @@ class StaleElementReferenceError < WebDriverError; end # 10
       class ElementNotVisibleError < WebDriverError; end # 11
 
       #
-      # Raised when an interaction could not be performed because the element
-      # is in an invalid state (e.g. attempting to click a disabled element).
+      # A command could not be completed because the element is in an
+      # invalid state, e.g. attempting to click an element that is no
```

Comment:
Good point - https://github.com/w3c/webdriver/issues/989 We'll see what is said.